docs-chef-io/content/inspec/resources/aws_api_gateway_v2_api.md

```diff
@@ -98,6 +98,11 @@ end
 `version`
 : A version identifier for the API.
 
+`failonwarnings`
+: Specifies to rollback the API creation when a warning is encountered. By default, API creation continues if a warning is encountered.
+
+: **Field**: `failonwarnings`
```

Comment:
Please remove this. There is no attribute named failonwarnings. 

https://docs.aws.amazon.com/sdk-for-ruby/v3/api/Aws/ApiGatewayV2/Client.html#get_api-instance_method